resources/install.sh

```diff
@@ -34,6 +34,12 @@ cd "$INSTALL_DIR/marketplace"
 # Remove old custom app name if exists
 spicetify config custom_apps spicetify-marketplace-
 
+# Color injection fix
+spicetify config inject_css 1
+spicetify config replace_colors 1
+spicetify config current_theme SpicetifyDefault
```

Comment:
Is there a way to check if they have the colour scheme set? People will get confused if they install marketplace and then lose their theme.

Reply:
Seems like it is possible, will patch in a sec.
![Uploading image.png…]()

Reply:
@kyrie25 I don't run windows unfortunately.